docs/tutorial/mac-app-store-submission-guide.md

```diff
@@ -1,68 +1,105 @@
+certicate can not run directly, it must be also be signed by Apple to be able to
+run, which will only be possible after downloading from Mac App Store.
 
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-  <dict>
-    <key>com.apple.security.app-sandbox</key>
-    <true/>
-    <key>com.apple.security.inherit</key>
-    <true/>
-  </dict>
-</plist>
-```
+#### Other Certificates
+
+You may notice that there are also other kinds of certificates.
+
+The "Developer ID Application" certificate is used to sign apps before
+distributing it outside the Mac App Store.
+
+The "Developer ID Installer" and "Mac Installer Distribution" certificates are
+used to sign the Mac Installer Package instead of the app itself. Most Electron
+apps do not use Mac Installer Package so they are generally not needed.
+
+Apps signed with "Apple Development" and "Apple Distribution" certificates can
+only run under sandbox mode, so they must use the MAS build of Electron. But
+the "Developer ID Application" certificate does not have this restrictions, apps
+signed with it can use either the normal build or the MAS build of Electron.
+
+#### Legacy Certificate Names
+
+Apple has been changing the names of certificates during past years, you might
+encounter them when reading old documentations, and some utilities are still
+using one of the old names.
+
+* The "Apple Distribution" certificate was also named as "3rd Party Mac
+  Developer Application" and "Mac App Distribution".
+* The "Apple Development" certificate was also named as "Mac Developer" and
+  "Development".
+
+### Install Provisioning Profile
+
+If you are going to submit the app to Mac App Store, to test it on your local
+machine before submitting it, you have to install the app's provisioning profile
+on your machine, and sign the app with the "Apple Development" certificate and
+the provisioning profile.
+
+To register the provisioning profile, you can follow below steps:
+
+1. Open the "Certificates, Identifiers & Profiles" page on the
+   [Apple Developer](https://developer.apple.com/account) website.
+2. Add a new App ID for you app in the "Identifiers" page.
+3. Register your local machine in the "Devices" page, you can find your
+   machine's "Device ID" in the "Hardware" page of the "System Information" app.
+4. Register a new Provisioning Profile in the "Profiles" page, and download it
+   to `/path/to/yourapp.provisionprofile` and install it.
 
-`parent.plist`:
+### Enable Sandbox in Apps
+
+Apps submitted to Mac App Store must run under [App Sandbox][app-sandboxing],
+and only the MAS build of Electron can run under sandbox mode.
+
+When signing app with `electron-osx-sign`, it will automatically add App Sandbox
+capacity to app's entitlements, but if you are using custom entitlements
+following capacities must be added:
```

Comment:
Added a few more words here to make the "App Sandbox" branding explicit everywhere, to avoid any possibility of confusion with other uses of `sandbox` in Electron.

```suggestion
### Enable Apple's App Sandbox

Apps submitted to the Mac App Store must run under Apple's
[App Sandbox][app-sandboxing]. Only the MAS build of Electron
can run in the App Sandbox.

When signing your app with `electron-osx-sign`, it will automatically add the App
Sandbox capacity to your app's entitlements, but if you are using custom entitlements,
the following capacities must be added:
```